### ISSUE TITLE
crop command ?

### ISSUE THREAD

**Issue by lesion** [fluent-ffmpeg/node-fluent-ffmpeg#359]
Hi !
Would be great to have a crop method.
I'm thinking on implementing it myself. Anyone interested or already done it ?
Thanks

**Comment by njoyard** [comment]
That would be nice, would get along very well with `.size()`, `.aspect()` and `.autoPad()` :)

**Comment by toxict** [comment]
Is there a way to crop thumbnails using the videoFilters method?

**Comment by njoyard** [comment]
No, it's not possible currently. Thumbnails generation uses a complex filtergraph, and ffmpeg does not support both video filters and a complex filtergraph on the same command. Plus fluent-ffmpeg currently does not allow setting multiple complex filtergraphs.

I'd like to fix some things in the thumbnails recipe in the future, namely adding support for filters and maybe enabling setting crop options when calling it.

**Comment by toxict** [comment]
Cropping thumbnails is an important feature along with applying generic filters to the thumbnail command.  Can you provide an ETA for when you'll implement the feature or provide some pointers for us to implement it ourselves and contribute the feature.

**Comment by njoyard** [comment]
I don't have an ETA for you, not enough free time for now.
Feel free to look at the code (lib/recipes.js) and contribute. I'll be happy to review or answer any of your questions.

**Comment by serg1uu** [comment]
Can you please tell us this feature's status?

**Comment by Sectaur** [comment]
Any update on this feature ? It would be very well received ! :)
